deployment/v3/mosip/partner-onboarder/README.md

```diff
@@ -4,6 +4,14 @@
 Loads certs for default partners for sandbox. Refer [mosip-onboarding repo](https://github.com/mosip/mosip-onboarding).
 
 ## Install 
+* Create a directory for onboarder on the NFS server at `/srv/nfs/<sandbox>/onboarder/`:
```

Comment:
and make sure it is added correctly in exports file of nfs server.

Reply:
added